pkg/multicluster/razee/razee_cluster_manager.go

```diff
@@ -70,16 +73,16 @@ func (r *razeeClusterManager) GetClusters() ([]multicluster.Cluster, error) {
 	for _, c := range razeeClusters {
 		resourceContent, err := r.con.Resources.ResourceContent(r.orgID, c.ClusterID, clusterMetadataConfigMapSL)
 		if err != nil {
-			r.log.Error().Err(err).Str("cluster", c.Name).Msg("Could not fetch cluster information")
+			r.log.Error().Err(err).Str(clusterKey, c.Name).Msg("Could not fetch cluster information")
 			return nil, err
 		}
 		// If no content for the resource was found the cluster is not part of Fybrik or not installed
 		// correctly. Fybrik should ignore those clusters and continue.
 		if resourceContent == nil {
-			r.log.Error().Err(err).Str("cluster", c.Name).Msg("Resource content returned is nil! Skipping cluster")
+			r.log.Error().Err(err).Str(clusterKey, c.Name).Msg("Resource content returned is nil! Skipping cluster")
 			continue
 		}
-		scheme := runtime.NewScheme()
+		scheme = runtime.NewScheme()
```

Comment:
@roytman 
I'm not sure if the re-assignment of `scheme` would  create a problem here